rust/protocol/src/protocol.rs

```diff
@@ -79,7 +89,7 @@ impl SignalMessage {
         receiver_identity_key: &IdentityKey,
     ) -> Result<Self> {
         let message = proto::wire::SignalMessage {
-            ratchet_key: Some(sender_ratchet_key.serialize().into_vec()),
+            ratchet_key: Some(sender_ratchet_key.serialize().to_vec()),
```

Comment:
Don't forget to take these back out when you put the variable-length PublicKey/PrivateKey API back.